src/analyze/analyze.c

```diff
@@ -953,7 +953,7 @@ static int analyze_critical_chain(sd_bus *bus, char *names[]) {
         if (n <= 0)
                 return n;
 
-        h = hashmap_new(&string_hash_ops);
+        h = hashmap_new(&string_hash_ops, NULL);
```

Comment:
hmm, maybe instead of adding this new parameter to all invocations of set_new() and hashmap_new() it might be nicer to add hashmap_new_full() or so, that takes the additional parameter, and then make hashmap_new() a macro that wraps it and passes the second parameter as NULL? Keeps things more readable for the common case I think, and makes the patch a good chunk smaller?